.gitignore

```diff
@@ -16,3 +16,4 @@
 configure.wrf*
 *.backup
 *.f90
+!chem/KPP/kpp/kpp-2.1/**/*.f90
```

Comment:
@zxdawn 
Please explain the semantics and syntax in this file. For example, 
1. what is the impact of two rules governing the *.f90 files
2. what do two `**` together signify

Reply:
@davegill `*.f90` means ignore all `.f90` files. After `!chem/KPP/kpp/kpp-2.1/**/*.f90` is added, .f90 files under all subdirectories of `chem/KPP/kpp/kpp-2.1/` will be kept.

I have checked all names of .f90 files under these subdirectories after the compilation of WRF-Chem.

They are as same as these stored in GitHub now. So, this method could keep necessary .f90 files for KPP and ignore many .f90 files generated after compilation of WRF.